### PR TITLE
feat: generate index.json

### PR DESCRIPTION
**Pull request by olenagerasimova**
part of #221 
Implemented main workflow to add/update index.json. Only one test is added for now, more tests will be added later...

### CODE REVIEW

**Comment by olenagerasimova** [comment]
@dgarus please, reaview

**Comment by olenagerasimova** [comment]
@dgarus please, review

File: src/main/java/com/artipie/nuget/IndexJson.java [hunk 1]
```diff
@@ -62,9 +98,94 @@ public Update(final InputStream input) {
...
         JsonObject perform(final NuGetPackage pkg) {
-            throw new NotImplementedException("Not implemented yet");
+            final JsonObjectBuilder res = Json.createObjectBuilder();
+            final Nuspec nuspec = pkg.nuspec();
+            final JsonObject newest = newPackageJsonItem(nuspec);
+            final String version = nuspec.version().normalized();
+            final JsonArrayBuilder items = Json.createArrayBuilder();
+            final String lower;
+            final String upper;
+            if (this.input.isPresent()) {
```
Comment (dgarus) [inline src/main/java/com/artipie/nuget/IndexJson.java:111]: We can make this method shorter by removing `lower` and `upper` variables.

**Review by dgarus — CHANGES_REQUESTED** [review]
@olenagerasimova thanks!
Minor comment.

**Comment by olenagerasimova** [comment]
@dgarus thanks, corrected

**Review by dgarus — APPROVED** [review]
Thanks!
LGTM

**Pull request merged**